package.json

```diff
@@ -64,18 +64,11 @@
     }
   ],
   "jspm": {
-    "directories": {
-      "lib": "dist"
-    },
     "main": "dist/goldenlayout",
-    "files": [
-      "dist/goldenlayout.js",
-      "dist/goldenlayout.min.js"
-    ],
     "format": "global",
     "registry": "jspm",
     "dependencies": {
-      "jquery": "*"
+      "jquery": "^2.1.0"
```

Comment:
@hoxton-one I set it as `^2.1.0` because the `package.json` of version 1.0.8 had `2.1.1` as a dependency: https://github.com/hoxton-one/golden-layout/blob/v1.0.8/package.json#L27-29.

However, I have no problem leaving the jquery version open, e.g. *****.

I just looked at Golden-Layout v1.0.9, however, and noticed you did indeed leave it as `^2.1.0`.  Was that intentional?